docs/docs/workers.md

```diff
@@ -63,7 +63,7 @@ In addition to `--burst`, `rq worker` also accepts these arguments:
 * `--connection-class`: Redis connection class to use, defaults to `redis.StrictRedis`.
 * `--log-format`: Format for the worker logs, defaults to `'%(asctime)s %(message)s'`
 * `--date-format`: Datetime format for the worker logs, defaults to `'%H:%M:%S'`
-
+* `--disable-job-desc-logging`: Toggle logging of a job description, defaults to True.
```

Comment:
Could you please update the explanation to something like `Turn off job description logging` or something similar? It's only a flag, so there's no default value.